tests/smoke/test.cli.build.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import {spawn} from 'child_process';
+import {describe, it} from 'mocha';
+
+import {
+  webExt, addonPath, artifactsPath,
+} from './common';
+
+describe('web-ext build', () => {
+  it('webext build --source-dir SRCDIR --artifacts-dir OUTDIR', () => {
```

Comment:
Sticking to [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development) makes things nice to read. Ideally, we should describe the behavior of `web-ext build` that we are testing and write it like a sentence in the call to `it(...)`. Does that pose any problems for the functional suite? I think something like it('builds and saves to a custom directory') would make sense here.